CHANGELOG.md

```diff
@@ -25,6 +25,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.0] - 2023-03-09
+### Changed
+- Launch the product-deletion-utility container for delete and uninstall actions.
+- Launch the <product>-install-utility container for activate actions.
+- Use the csm-docker-sle:15.4 image.
+### Added
+- Add the 'delete' action.
```

Comment:
Suggest adjusting spacing for consistency. Also suggest saying delete is an alias of uninstall.

```suggestion
- Use the csm-docker-sle:15.4 image.

### Added
- Add the 'delete' action as an alias to 'uninstall'

```